src/app/institutional/pages/institutional/institutional.component.ts

```diff
@@ -0,0 +1,137 @@
+import { Component, Inject, OnInit } from '@angular/core'
+import { WINDOW } from '../../../cdk/window'
+import { FormControl, FormGroup, Validators } from '@angular/forms'
+import { Observable } from 'rxjs'
+import { map, startWith, take } from 'rxjs/operators'
+import { DiscoService } from '../../../core/disco/disco.service'
+import { Institutional } from '../../../types/institutional.endpoint'
+import { CookieService } from 'ngx-cookie-service'
+import { environment } from '../../../../environments/environment.local'
```

Comment:
Oh sorry just found this one. Remember to import `environments/environment` or it will not change on qa/prod build

Reply:
Good catch! Fixed